x-pack/plugins/security_solution/public/detections/components/rules/schedule_item_form/index.tsx

```diff
@@ -49,12 +51,24 @@ const StyledEuiFormRow = styled(EuiFormRow)`
   max-width: none;
 
   .euiFormControlLayout {
-    max-width: 200px !important;
+    max-width: ${({ fullWidth }) => (fullWidth ? 'auto' : '200px !important')};
```

Comment:
Currently, ScheduleItem prop looks like so:
![image](https://user-images.githubusercontent.com/5354282/188847418-b5ed8c6c-0309-4ed5-9019-e03ea763e3e1.png)
Updating the styling when `fullWidth` prop is passed to match design:
https://user-images.githubusercontent.com/81987435/145276050-69d2c287-c22d-466d-9717-e056ab9180ba.png

Reply:
Can it be always `auto` in this component? Width could be set to `200px` for a parent container element used only on the Creation and Editing pages.

Reply:
Updated so that width is auto by default. Went to other use cases of `ScheduleItem` and set the max-width to 200px to its parent container, where needed.

Only exception is in a usage within the form for creating a rule of type `New terms`, for the "History Window Size" field:

**Before:** 
![image](https://user-images.githubusercontent.com/5354282/189136411-5aec39fe-8e79-4cdf-9ff0-13244df4a9da.png)

**After:**
![image](https://user-images.githubusercontent.com/5354282/189136506-a51fbbd3-6cd4-4904-aee0-ce2f08303e1e.png)

which I think actually improves the UI.